documentation/modules/configuring/proc-managing-node-pools-ids.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='proc-managing-node-pools-ids-{context}']
+= (Preview) Assigning IDs to node pools for scaling operations
+
+[role="_abstract"]
+This procedure describes how to use annotations for advanced node ID handling by the Cluster Operator when performing scaling operations on node pools. 
+You specify the node IDs to use, rather than the Cluster Operator using the next ID in sequence.
+Management of node IDs in this way gives greater control.
+
+The steps assume that you are using ZooKeeper for cluster management. 
+The operation is not possible for clusters using KRaft mode.
+
+To add a range of IDs, you assign the following annotations to the `KafkaNodePool` resource:
+
+* `strimzi.io/next-node-ids` to add a range of IDs for scaling up
+* `strimzi.io/remove-node-ids` to add a range of IDs for scaling down
```

Comment:
As mentioned above, this might make it more clear it applies to any created / removed broker.

```suggestion
* `strimzi.io/next-node-ids` to add a range of IDs that will be used for new brokers
* `strimzi.io/remove-node-ids` to add a range of IDs for removing existing brokers
```